### PR TITLE
Use asciidoc property to specify spring boot version in Initializr

### PR DESCRIPTION
**Pull request by ilayaperumalg**

### CODE REVIEW

File: spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc [hunk 1]
```diff
@@ -1538,7 +1538,9 @@ For Spring Cloud Stream samples, please refer to the https://github.com/spring-c
 
 == Getting Started
 
-To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".  Note that Spring Cloud Stream applications are known to work with Spring Boot versions from 1.3.x to {supported-spring-boot-version}. In the _Search for dependencies_ text box type `Stream Rabbit` or `Stream Kafka` depending on what binder you want to use.
+To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".
+Select Spring Boot {supported-spring-boot-version} in the dropdown.
```
Comment (markpollack) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: you need to define this variable in index.adoc

Comment (ilayaperumalg) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: It is from [here] (https://github.com/spring-cloud/spring-cloud-stream/pull/677/files#diff-c289cb8e00827701d86a1312504841a3R106)

Comment (markpollack) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: I see, I guess this works as we will likely release against latest boot versions.

Comment (ilayaperumalg) [inline spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc]: yeah, also we know we already worked with that boot version.

**Comment by sobychacko** [comment]
LGTM - merged.

**Pull request closed without merging**